client/src/route/auth/AccountTabs.tsx

```diff
@@ -1,21 +1,80 @@
 import * as React from 'react';
 import TabComponent from 'components/tab/TabComponent';
 import { TabData } from 'components/tab/subcomponents/TabRender';
-
-const accountTab: TabData[] = [
-  {
-    label: 'Profile',
-    body: <>Profile - potentially visible to other users.</>,
-  },
-  {
-    label: 'Settings',
-    body: <>Account settings - private to a user.</>,
-  },
-];
+import { useAuth } from 'react-oidc-context';
 
 const scope: TabData[][] = [];
 
 function AccountTabs() {
+  const auth = useAuth();
+  const groups: string[] = (auth.user?.profile.groups_direct as string[]) || [];
+
+  if (groups.length === 0) {
+    const accountTab: TabData[] = [
+      {
+        label: 'Profile',
+        body: (
+          <>
+            Profile - potentially visible to other users.
+            <br></br>
+            <img
+              src={auth.user?.profile.picture}
+              style={{ height: '5%', width: '5%' }}
+            ></img>
+            Username is <b>{auth.user?.profile.preferred_username}</b>
+            <br></br>
+            See more details on{' '}
+            <a href={auth.user?.profile.profile}>SSO OAuth provider.</a>
+          </>
```

Comment:
Please add the groups text and code here. The exact text is available in issue #478

Reply:
Have added it in the conditional when the user is part of group/proups, this is tge the accounTabs constant content when he is not part of any of them